src/components/LanguageSelection/LanguageSelection.stories.mdx

```diff
@@ -33,7 +33,6 @@ export const ConfigDecorator = (Story, { args }) => (
         type: { summary: 'number' },
       },
     },
-    onLanguageChanged: { table: { disable: true } },
```

Comment:
this arg is not relevant for the `LanguageSelectionDisplay` component and you can't change it in the controls anyway, that's why I disabled it for the canvas.